tests/common/dualtor/data_plane_utils.py

```diff
@@ -188,7 +201,8 @@ def send_t1_to_server_with_action(duthosts, ptfhost, ptfadapter, tbinfo):
     
     duthosts_list = []
     def t1_to_server_io_test(activehost, tor_vlan_port=None,
-                            delay=0, action=None, verify=False, send_interval=None):
+                            delay=0, action=None, verify=False, send_interval=None,
+                            stop_after=0):
```

Comment:
Can we make the default `None` or `-1`? In case we do ever have a scenario where we want to stop traffic immediately after the callback action.

Reply:
Changed to `None`.